virtual_database_client/db_client.go

```diff
@@ -0,0 +1,316 @@
+// Data client for new Virtual Path
+
+package client
+
+import (
+    //"bytes"
+    //"encoding/json"
```

Comment:
Please clean this.

Reply:
Ok. Personally I found all these commented out imports very helpful. But I will remove them as requested.